lib/clickUpHelpers.js

```diff
@@ -0,0 +1,37 @@
+// Third-party dependencies
+const axios = require('axios')
+
+// In-house dependencies
+const helpers = require('./helpers')
+
+const clickUpTeamName = helpers.getEnvVar('CLICKUP_TEAM_NAME')
+const clickUpTeamId = helpers.getEnvVar('CLICKUP_TEAM_ID')
+
+// Middleware function to check for users logged into Brave's ClickUp
+async function clickUpChecker(req, res, next) {
```

Comment:
This was your clickUpValidator function, transformed into Express middleware.

Reply:
Looks great.